api/tests/unit/domain/usecases/accept-certification-center-invitation_test.js

```diff
@@ -1,15 +1,20 @@
-const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const { expect, sinon, domainBuilder, catchErr } = require('../../../test-helper');
 const acceptCertificationCenterInvitation = require('../../../../lib/domain/usecases/accept-certification-center-invitation');
 const CertificationCenterInvitedUser = require('../../../../lib/domain/models/CertificationCenterInvitedUser');
 const CertificationCenterInvitation = require('../../../../lib/domain/models/CertificationCenterInvitation');
+const { AlreadyExistingEntityError } = require('../../../../lib/domain/errors');
 
 describe('Unit | UseCase | accept-certification-center-invitation', function () {
-  it('should return the user invitation', async function () {
+  it('should get user invitation', async function () {
```

Comment:
**remarque**
Je comprenais mieux la formulation avec `return the user invitation` plutôt que `get user invitation` 🤔